buildtest/cli/report.py

```diff
@@ -7,27 +7,27 @@
 from buildtest.defaults import BUILD_REPORT, BUILDTEST_REPORTS, console
 from buildtest.exceptions import BuildTestError
 from buildtest.utils.file import is_file, load_json, resolve_path
-from rich.color import ANSI_COLOR_NAMES, Color, ColorParseError
+from rich.color import Color, ColorParseError
 from rich.table import Table
 
 logger = logging.getLogger(__name__)
 
 
 def checkColor(colorArg):
     """Checks the provided colorArg against the compatible colors from Rich.Color"""
-    if type(colorArg) is Color:
+    if isinstance(colorArg, Color):
         return colorArg.name
 
-    if colorArg and type(colorArg) is list:
+    if colorArg and isinstance(colorArg, list):
         colorArg = colorArg[0]
         return colorArg
-    if type(colorArg) is str:
+    if isinstance(colorArg, str):
         try:
             checkedColor = Color.parse(colorArg).name
         except ColorParseError:
             checkedColor = Color.default().name
         return checkedColor
-    return None
+    return
```

Comment:
you dont need this `return` statement since this is the last line in method and python will return from method. Only use `return` when needed specially when one needs to return out of method using an if condition.

Reply:
removed